app/scripts/components/exploration/components/datasets/data-layer-card.tsx

```diff
@@ -155,16 +158,16 @@ export default function DataLayerCard(props: CardProps) {
   });
 
   const showLoadingConfigurableCmapSkeleton =
-    showConfigurableColorMap &&
-    dataset.status === 'loading' &&
-    datasetLegend?.type === 'gradient';
+    dataset.status === 'loading' && datasetLegend?.type === 'gradient';
   const showConfigurableCmap =
-    showConfigurableColorMap &&
     dataset.status === 'success' &&
+    dataset.data.type !== 'wmts' &&
+    dataset.data.type !== 'wms' &&
     datasetLegend?.type === 'gradient' &&
     colorMap;
+
   const showNonConfigurableCmap =
-    !showConfigurableColorMap && datasetLegend?.type === 'gradient';
+    !showConfigurableCmap && datasetLegend?.type === 'gradient';
```

Comment:
I assumed that this was a mistake? I think this behavior makes more sense, but I might be missing something.